### ISSUE TITLE
Enable Github Packages 

### ISSUE THREAD

**Issue by cdr-chakotay** [transloadit/java-sdk#55]
Implement a Workflow to release also on Github Packages besides Maven Central Repo.
This is a long scope aim

**Comment by Acconut** [comment]
Good idea to have Github Packages as a backup. Do you know if Github Packages is fully free for open-source projects?

**Comment by cdr-chakotay** [comment]
> Good idea to have Github Packages as a backup. Do you know if Github Packages is fully free for open-source projects?

I don't know which plan we are currently using, but in total free payment plan at least 500 MB are free. If we are in the team plan with Transloadit it is 2 GB. But it seems to be entirely free for public repos :) 
[https://github.com/pricing](https://github.com/pricing)

**Comment by Acconut** [comment]
>  But it seems to be entirely free for public repos :)

That's good!
